kubernetes/k8s-listener.py

```diff
@@ -388,6 +444,78 @@ def main():
         if c != '\r' or c2 != '\n':
             raise "Expected CRLF, got %c%c" % (c, c2)
 
+def get_romana_hosts():
+    """
+    returns a list of romana host IPs
+    
+    Example: ["10.0.0.1", 10.1.0.1" ]
+    """
+
+    romana_hosts = []
+    r = requests.get(topology_url)
```

Comment:
Might be a good idea to check the status code of the result to see that we actually got something.